update.sh

```diff
@@ -104,25 +109,24 @@ echo '#### DO NOT MODIFY. THIS FILE IS AUTOGENERATED ####
 ' | cat - travis.yml.template >.travis.yml
 
 for version in "${versions[@]}"; do
-  # Skip "docs" and other non-docker directories
-  [ -f "${version}/Dockerfile" ] || continue
-
   parentpath=$(dirname "${version}")
   versionnum=$(basename "${version}")
   baseuri=$(get_config "${parentpath}" "baseuri")
   update=$(in_versions_to_update "${version}")
 
-  add_stage "${baseuri}" "${version}" "default"
-
-  if [ "${update}" -eq 0 ]; then
-    info "Updating version ${version}..."
-    update_node_version "${baseuri}" "${versionnum}" "${parentpath}/Dockerfile.template" "${version}/Dockerfile" &
-  fi
-
   # Get supported variants according the target architecture
   # See details in function.sh
   IFS=' ' read -ra variants <<<"$(get_variants "${parentpath}")"
 
+  if [ -f "${version}/Dockerfile" ]; then
+    add_stage "${baseuri}" "${version}" "default"
+
+    if [ "$update" -eq 0 ]; then
+      info "Updating version $version..."
```

Comment:
`{ }` missing, original line 117/118 also use it.

Reply:
Done!